src/report/generate_trustymail_report.py

```diff
@@ -1114,8 +1116,8 @@ def process_record(domain, record, policy_published):
                 dkim = auth_results["dkim"]
                 dkim_and_alignment = policy_evaluated["dkim"]
                 if isinstance(dkim, list):
-                    x["DKIM Result"] = " ".join([y["result"] for y in dkim])
-                    x["DKIM Domain"] = " ".join([y["domain"] for y in dkim])
+                    x["DKIM Result"] = " ".join([str(y["result"]) for y in dkim])
+                    x["DKIM Domain"] = " ".join([str(y["domain"]) for y in dkim])
```

Comment:
Generator usage again.

```suggestion
                    x["DKIM Result"] = " ".join(str(y["result"]) for y in dkim)
                    x["DKIM Domain"] = " ".join(str(y["domain"]) for y in dkim)
```